modules/custom/az_dam/config/quickstart/core.entity_view_display.media.ib_dam_embed.default.yml

```diff
@@ -0,0 +1,41 @@
+uuid: b2f16ee4-7096-4d1c-a77e-43a1185a9475
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.media.ib_dam_embed.field_media_ib_dam_embed
+    - media.type.ib_dam_embed
+  module:
+    - ib_dam
+_core:
+  default_config_hash: rY3bCqMOFUDYqHzQJKkJQ23jkrFvgOAbuvtzkTzHha0
```

Comment:
```suggestion
```

Reply:
Resolved locally. Will commit for review when all changes have been made.